backend/logic/courses/courses.js

```diff
@@ -145,9 +147,29 @@ var addCourseOfferings = async courseOfferings => {
   }
 };
 
+let phaseOutCourse = async courseCode => {
+  let connection = await mysql.getNewConnection();
+  try {
+    if (!courseCode) {
```

Comment:
check format of course code? format.js might already have a function for that, otherwise could write a simple one

Reply:
You can use **verifyCourseCode()** which just got merged into dev